### DISCUSSION TITLE
No path to getting "Demolitionist" talent

### DISCUSSION THREAD

**Discussion by ww123ist** [bcholmes/StarTrek2d20#204] [General]
I tried making a character sheet now that the Federation-Klingon War update was given, but I realized that there is no mathematical way to apply the Demolitionist talent as there is no way for the current program to have Engineering 5 and Security 3 by the time you pick all four talents. 

Could you possibly add a double-check feature at the end to review talents, the same way you do values? It might also help to do the same for focuses while you're at it. 

If this doesn't seem easy, perhaps you could make it to where you have the "allow cross-species" and "esoteric" talents to also have an option to void requirements of department or attribute levels. It would purely be on an honor code basis, but some players or DMs might not want to stand on ceremony anyhow.

**Comment by Felderburg** [comment]
This is similar to #56, in that it also involves talents and their requirements being an issue. I also suggested a check at the end, although for the opposite reason (a talent can be taken before adding to disciplines that make it unallowed). I definitely agree that a 'use any and all talents' button would be nice, especially paired with a checker at the end.

**Comment by ww123ist** [comment]
I offer a new idea: perhaps you could make it so that the talents are entered into an editable textbox rather than the current unchangeable format they're in now, similar to the values section. This would allow us to write in and edit talents any time, especially for level ups and choosing to add them in or exchange them -- I know I have to remake a character sheet using your generator every time I want to exchange talents or make slight edits involving them, rather than just working on an existing sheet. I know the only challenge is the fact that there is such a wide array of text length, but I think it's an idea worth at least testing!
